cogs/assistance-cmds/boot3dsx.3ds.md

```diff
@@ -3,4 +3,8 @@ help-desc: Download link for 3DS Homebrew Launcher, boot.3dsx
 aliases: hbl
 ---
 
-The 3DS Homebrew Launcher, [boot.3dsx](https://github.com/devkitPro/3ds-hbmenu/releases/download/v2.4.1/boot.3dsx)
+The 3DS Homebrew Launcher, download the latest release zip from here:
```

Comment:
If we're going this route, then we should add a period and start a new sentence for instructions